src/main/scala/org/globalforestwatch/ForestChangeDiagnostic.scala

```diff
@@ -101,25 +100,25 @@ object ForestChangeDiagnostic extends SparkCommand  {
 
 
     val analyses = List(
-      Analysis.TreeCoverLossTotalYearly,
-      Analysis.TreeCoverLoss90Yearly,
-      Analysis.TreeCoverLossPrimaryForestYearly,
-      Analysis.TreeCoverLossPeatYearly
+      Analysis.TreeCoverLossTotalYearlyPW,
+      Analysis.TreeCoverLoss90YearlyPW,
+      Analysis.TreeCoverLossPrimaryForestYearlyPW,
+      Analysis.TreeCoverLossPeatYearlyPW
```

Comment:
I switched nomenclature here to indicate that these analysis functions are to be applied pixelwise.